src/components/dialogs/network-modifications/hvdc-line/lcc/creation/lcc-converter-station.tsx

```diff
@@ -0,0 +1,242 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { FloatInput, MODIFICATION_TYPES, TextInput } from '@gridsuite/commons-ui';
+import {
+    BUS_OR_BUSBAR_SECTION,
+    CONNECTED,
+    CONNECTION_DIRECTION,
+    CONNECTION_NAME,
+    CONNECTION_POSITION,
+    CONNECTIVITY,
+    CONVERTER_STATION_ID,
+    CONVERTER_STATION_NAME,
+    FILTERS_MCS_TABLE,
+    ID,
+    LOSS_FACTOR,
+    MAX_Q_AT_NOMINAL_V,
+    MCS_SELECTED,
+    POWER_FACTOR,
+    SHUNT_COMPENSATOR_ID,
+    SHUNT_COMPENSATOR_NAME,
+    VOLTAGE_LEVEL,
+} from '../../../../../utils/field-constants';
+import { percentageTextField, sanitizeString } from '../../../../dialog-utils';
+import { CurrentTreeNode } from '../../../../../../redux/reducer';
+import { UUID } from 'crypto';
+import { ConnectivityForm } from '../../../../connectivity/connectivity-form';
+import { Grid } from '@mui/material';
+import useVoltageLevelsListInfos from '../../../../../../hooks/use-voltage-levels-list-infos';
+import GridSection from '../../../../commons/grid-section';
+import GridItem from '../../../../commons/grid-item';
+import yup from '../../../../../utils/yup-config';
+import {
+    getConnectivityFormData,
+    getConnectivityWithPositionEmptyFormData,
+    getConnectivityWithPositionSchema,
+} from '../../../../connectivity/connectivity-form-utils';
+import FiltersMcsTable from './filters-mcs-table';
+import {
+    Connectivity,
+    FilterMcsTable,
+    LccConverterStationFormInfos,
+    LccConverterStationInfos,
+    McsOnSide,
+} from './lcc-creation.type';
+import { UNDEFINED_CONNECTION_DIRECTION } from '../../../../../network/constants';
+
+export const getLccConverterStationSchema = () =>
+    yup.object().shape({
+        [CONVERTER_STATION_ID]: yup.string().nullable().required(),
```

Comment:
```suggestion
        [CONVERTER_STATION_ID]: yup.string().required(),
```
I don't see any benefit of using nullable following with required(), to remove if possible to reduce the complexity in other codes